Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -31,12 +31,13 @@ public class DependencyTrackingTelemetryModule : ITelemetryModule, IDisposable
 
 #if NET45 || NETCORE
         private HttpCoreDiagnosticSourceListener httpCoreDiagnosticSourceListener;
+        private SqlClientDiagnosticSourceListener sqlClientDiagnosticSourceListener;
```

Comment:
is SqlClientDiagnosticSourceListener present in .net core as well as full .net? I could not find any logic to prevent collision/duplication when Profiler is present as well.